### DISCUSSION TITLE
Add file pathname to info inspector

### DISCUSSION THREAD

**Discussion by pravdomil** [CommandPost/CommandPost#3232] [Technical Support] [answered]
Based on `/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/Metadata.framework/Versions/A/Resources/en.lproj/schema.strings` there is
```
	kMDItemPath = "File pathname";
	kMDItemPath.Description = "Complete pathname of this file";
	kMDItemPath.ShortName = "path";
```
Is there any way to get it into info inspector?

Metadata view fields is stored in `~/Library/Preferences/com.apple.FinalCut.plist` under `metadataViews` key.

**Comment by latenitefilms** [comment, marked as answer]
You can only add the Spotlight properties that are accessible from the Inspector:

<img width="743" alt="image" src="https://github.com/CommandPost/CommandPost/assets/22286696/c10ba41f-dfef-486c-840b-a85a0474cb0a">

I can't see anything relevant in terms of file path.

If you want to get the file path of the currently selected file, the only way I can think of doing this is:

1. Triggering **File > Reveal In Finder** then copying the path from Finder
2. Triggering **Edit > Copy**, then reading the Pasteboard data to get the file path, keeping in mind that the Pasteboard data structure could change between versions.
3. Build some kind of user interface, so that you drag the clip from the Timeline or Browser to your user interface (which sends `FCPXML`), and then read the `FCPXML` contents.
